src/js/debug.js

```diff
@@ -754,6 +754,7 @@ const DEBUG = {
             'debug[1]': 'Adjusted Throttle',
             'debug[2]': 'Upper Limit',
             'debug[3]': 'EZ Land Limit',
+            'debug[4]': 'Speed Limit',
```

Comment:
Due to reasons (I was tired), I think the speed ended up at debug[5], and I think it might stay there, the stick deflection limit should probably have its own field too at some point in time.

Reply:
@tbolin :grin: https://github.com/betaflight/betaflight/pull/13411#discussion_r1510079932

Reply:
debug[4] is now Stick limit, debug[5] is still Speed limit